scripts/globals/synergy.lua

```diff
@@ -0,0 +1,267 @@
+-----------------------------------
+-- Synergy
+-----------------------------------
+xi = xi or {}
+xi.synergy = xi.synergy or {}
+
+-----------------------------------
+-- Synergy Furnace
+-----------------------------------
+
+-- NOTE: Change this to play with the synergy furnace
+local TEST_SYNERGY = false
+
+local getBytes = function(value)
+    local byte0 = bit.rshift(bit.band(value, 0x000000FF), 0)
+    local byte1 = bit.rshift(bit.band(value, 0x0000FF00), 8)
+    local byte2 = bit.rshift(bit.band(value, 0x00FF0000), 16)
+    local byte3 = bit.rshift(bit.band(value, 0xFF000000), 24)
+    return byte0, byte1, byte2, byte3
+end
+
+xi.synergy.synergyFurnaceOnTrade = function(player, npc, trade)
+    if not TEST_SYNERGY then
+        return
+    end
+
+    local arg0 = 8
+    local arg1 = 83886085 -- Galley Kitchen?
+    local arg2 = 0
+    local arg3 = 69121
```

Comment:
Item ID looks like it's the first 16 bits of arg3.

Reply:
Quite right, thanks!